client/tos/modal/index.js

```diff
@@ -30,13 +30,13 @@ const TosLink = ( props ) => (
 
 const TosModalUI = ( { onAccept, onDecline, isBusy, hasError } ) => {
 	const title = __(
-		'WooCommerce Payments: Terms of Service',
+		'WooCommerce Payments: Terms of service',
 		'woocommerce-payments'
 	);
 	const message = interpolateComponents( {
 		mixedString: __(
-			'To continue using WooCommerce Payments, please review and agree to our {{link}}Terms of Service{{/link}}.' +
-				' By clicking “Accept” you agree to the Terms of Service.',
+			'To continue using WooCommerce Payments, please review and agree to our {{link}}Terms of service{{/link}}.' +
+				' By clicking “Accept” you agree to the terms of service.',
```

Comment:
The second `terms of service` doesn't look consistent with the other ones. I'm adding this comment just so we can remember it after we confirm which capitalization best fits the sentence.